Configurations/windows-makefile.tmpl

```diff
@@ -632,10 +631,13 @@ reconfigure reconf:
       my ($gen0, @gens) = @{$args{generator}};
       my $generator = '"'.$gen0.'"'.join('', map { " $_" } @gens);
       my $generator_incs = join("", map { " -I \"$_\"" } @{$args{generator_incs}});
-      my $incs = join("", map { " -I \"$_\"" } @{$args{incs}});
+      my $incs = join("", map { " -I\"$_\"" } @{$args{incs}});
       my $defs = join("", map { " -D".$_ } @{$args{defs}});
       my $deps = @{$args{deps}} ?
-          '"'.join('" "', @{$args{generator_deps}}, @{$args{deps}}).'"' : '';
+          join(' ',
+               map { file_name_is_absolute($_) || ($_ =~ m|^../|) ? "\"$_\"" : $_ }
+               (@{$args{generator_deps}}, @{$args{deps}}))
+          : '';
```

Comment:
Errr...  why?

Reply:
This I had to do to work around a bug/limitation in `make.exe`'s handling of quoted filepaths. Please see the commit message in b61f4af8b377dba27090cd03c19e02d314b7faea.

Reply:
Hmmm, is C++Builder still a current product?  This should be reported to them in that case.  If not, I understand why we need to put in that fix...  but frankly, oh the ugliness we have to add :-/

Would you mind adding a short comment on the matter in this spot?  Git history is great, but there's nothing like more direct explanations.

Reply:
Or, since this happens in more than one spot, a comment before the function definition is fine as well.

Reply:
Added comment above function.

Reply:
Saw it.  Muuuuch better, thank you